htdocs/comm/propal/card.php

```diff
@@ -2343,6 +2344,10 @@
 
 			$parameters = array();
 			$reshook = $hookmanager->executeHooks('formAddObjectLine', $parameters, $object, $action); // Note that $action and $object may have been modified by hook
+		}elseif($action == 'editline')
+		{
+			$parameters = array();
+			$reshook = $hookmanager->executeHooks('formEditObjectLine', $parameters, $object, $action); // Note that $action and $object may have been modified by hook
```

Comment:
When we are in edit line mode, then part of code
// Form to add new line
is not output because the edit form is already managed into the printObjectLines (with several available triggers).
Do we really need to introduce this new hook ?